package.json

```
@@ -36,15 +36,16 @@
   },
   "homepage": "https://github.com/Half-Shot/matrix-appservice-discord#readme",
   "dependencies": {
+    "better-discord.js": "https://github.com/Sorunome/better-discord.js.git#b5a28499899fe2d9e6aa1aa3b3c5d693ae672117",
```

```suggestion
    "better-discord.js": "git://github.com/Sorunome/better-discord.js.git#b5a28499899fe2d9e6aa1aa3b3c5d693ae672117",
```

While Yarn seems to gladly accept `https` for git repository, other tools like yarn2nix which is used to generate the Nix package require the right protocol to be set to properly download the dependency.

([It otherwise tries to download the page, confusing it with a tarball](https://github.com/NixOS/nixpkgs/blob/b22ab697faf5b8140b5eb1952220532d88492669/pkgs/servers/matrix-appservice-discord/yarn-dependencies.nix#L676-L683))